pkg/jobs/jobspb/jobs.proto

```diff
@@ -127,6 +127,26 @@ message BackupDetails {
   // written, i.e. the URI the user provided before a chosen suffix was appended
   // to its path.
   string collection_URI = 8 [(gogoproto.customname) = "CollectionURI"];
+
+  // The following fields are used by scheduled backups to chain protected timestamp records.
+  // For more details about the chaining scheme refer to
+  // ScheduledBackupExecutionArgs.ChainProtectedTimestampRecords.
+
+  // ReleaseOnSuccessPTSRecord is the id of the PTS record that a full backup
+  // running as part of a schedule will release on successful completion.
+  bytes release_on_success_pts_record = 10 [
```

Comment:
I wonder if it'd be worthwhile to separate the PTS ID that we're updating and whether we're releasing it or updating it. May also be worth mentioning that this PTS record is "owned" by a schedule.

Reply:
Are you suggesting the same thing as @miretskiy i.e. consolidate this into a single field and have an enum to determine whether we are updating it or releasing it? We _should_ never be doing both in a single backup jobs' execution.

Reply:
Yep; I had not seen Yevgeniy's comment and was similarly wondering if that approach had merits. I don't think this is a hard blocker from me though.